src/main/resources/application.conf

```diff
@@ -105,5 +105,6 @@ userprofile {
 
 nih {
   whitelistFile = "users.txt"
-  rawlsGroupName = "dbGapAuthorizedUsers"
+  rawlsGroupName = "testRealm"
+//  rawlsGroupName = "dbGapAuthorizedUsers"
```

Comment:
I think we do. IIRC, `dbGapAuthorizedUsers` is something that should exist in dev since it was part of the NIH whitelist stuff.